### PR TITLE
Refactor/line item68 cases Biospecimen

### PR DESCRIPTION
**Pull request by ocshawn**
## Description
Fixing issues in jira: [PEAR-1694](https://gdc-ctds.atlassian.net/browse/PEAR-1694) item 8
Number of 508 fixes to Repository Page:
- [8 a ii add some aria labels and consistency to search bar clear areas](https://github.com/NCI-GDC/gdc-frontend-framework/commit/b125fc4ab5e16ecf66935e2d743cfc183af13391)
- [8 a iii add some accessibility to BioTree](https://github.com/NCI-GDC/gdc-frontend-framework/commit/b5705da2e55c3fbcd3c758c3aef769b0effdded3)

## Checklist

- [x] Added proper unit tests
- [x] Left proper TODO messages for any remaining tasks
- [x] Scanned for web accessibility with **aXe**, and mitigated or documented
      flagged issues

### CODE REVIEW

File: packages/portal-proto/src/features/facets/ExactValueFacet.tsx [hunk 1]
```diff
@@ -104,10 +104,9 @@ const ExactValueFacet: React.FC<ExactValueProps> = ({
         color="white"
         radius="xl"
         variant="transparent"
-        arial-label="remove value from filter"
         onClick={() => setValues(textValues.filter((i) => i !== x))}
       >
-        <CloseIcon size={10} />
+        <CloseIcon size={10} arial-label="remove value from filter" />
```
Comment (amlehman) [inline packages/portal-proto/src/features/facets/ExactValueFacet.tsx]: `aria-label` is misspelled here.

File: packages/portal-proto/src/features/facets/SetFacet.tsx [hunk 2]
```diff
@@ -29,14 +29,8 @@ const FACET_TO_MODAL = {
...
-    <ActionIcon
-      size="xs"
-      color="white"
-      radius="xl"
-      variant="transparent"
-      arial-label={`remove ${x} from filter`}
-    >
-      <CloseIcon size={10} />
+    <ActionIcon size="xs" color="white" radius="xl" variant="transparent">
+      <CloseIcon size={10} arial-label={`remove ${x} from filter`} />
```
Comment (amlehman) [inline packages/portal-proto/src/features/facets/SetFacet.tsx]: Same here.

File: packages/portal-proto/src/features/facets/ExactValueFacet.tsx (hunk 1 above)
Comment (ocshawn) [reply packages/portal-proto/src/features/facets/ExactValueFacet.tsx]:
```suggestion
        <CloseIcon size={10} aria-label="remove value from filter" />
```

File: packages/portal-proto/src/features/facets/SetFacet.tsx (hunk 2 above)
Comment (ocshawn) [reply packages/portal-proto/src/features/facets/SetFacet.tsx]:
```suggestion
      <CloseIcon size={10} aria-label={`remove ${x} from filter`} />
```

**Review by amlehman — APPROVED** [review]

**Comment by cemile-ctds** [comment]
Please add aria-hidden="true" to SVG search icon.

**Comment by ocshawn** [comment]
@cemile-ctds added aria-hidden="true" please re review

**Comment by cemile-ctds** [comment]
Looks good. Thank you, Shawn.

**Pull request merged**